packages/editor/src/pages/Editor/components/Backstage/MySolutions/index.tsx

```diff
@@ -83,12 +111,24 @@ class MySolutions extends React.Component<IProps> {
         {(!isSignedIn || gistMetadata.length > 0) && (
           <GalleryList
             title="My shared gists on GitHub"
-            items={gistMetadata.map(gist => ({
-              key: gist.id,
-              title: gist.title,
-              description: gist.description,
-              onClick: () => openGist(gist),
-            }))}
+            items={gistMetadata
+              .filter((meta: ISharedGistMetadata) => {
+                if (this.state.filterQuery === '') {
+                  return true;
+                }
+
+                const megastring = [meta.title, meta.description]
+                  .filter(Boolean)
```

Comment:
What the heck does `.filter(Boolean)` do?

Reply:
Boolean is technically a class constructor, or a function, and it will take whatever value you give it and give you true for a truthy value, and false for a falsey value